library/sync/src/test/kotlin/org/cru/godtools/sync/GodToolsSyncServiceTest.kt

```diff
@@ -19,30 +18,38 @@ import kotlinx.coroutines.test.TestScope
 import kotlinx.coroutines.test.UnconfinedTestDispatcher
 import kotlinx.coroutines.test.runCurrent
 import kotlinx.coroutines.test.runTest
-import org.cru.godtools.sync.task.BaseSyncTasks
 import org.cru.godtools.sync.task.ToolSyncTasks
+import org.cru.godtools.sync.task.UserCounterSyncTasks
 import org.cru.godtools.sync.work.scheduleSyncToolsWork
 import org.junit.After
+import org.junit.Assert.assertFalse
 import org.junit.Assert.assertTrue
 import org.junit.Before
 import org.junit.Test
-import org.junit.runner.RunWith
 import timber.log.Timber
 
-@RunWith(AndroidJUnit4::class)
```

Comment:
For my own clarification; why is this no longer needed?

Reply:
Uses the Android test runner loads the Android apis with robolectric. It adds about 10 seconds to test startup.

Robolectric is only necessary if something in the test or objects being tested references/uses an android api